content/news/029/index.md

```diff
@@ -190,6 +190,19 @@ Version 0.9.0 brings a few breaking changes. Notably `wgpu` was updated to
 [pixels]: https://github.com/parasyte/pixels
 [pixels-changelog]: https://github.com/parasyte/pixels/releases/tag/0.9.0
 
+### [bevy-remote-devtools]
+
+![Demo of the tauri based development app](bevy-remote-devtools.gif)
+
+[bevy-remote-devtools] is a plugin and UI application for the [Bevy] game engine allowing to view entities and their components, asset resources like meshes, events from the [tracing] crate and system timings using a very basic profiler. It also supports all of that over network so debugging can be done from any remote machine and vice versa.
+
+The first release of the 0.1 version comes with basic support for the above mentioned features. It contains a plugin for [Bevy] that will extend you application with a small REST HTTP API that can be consumed by the included [Tauri] based UI application.
```

Comment:
Instead of "above mentioned" I think it should be "aforementioned"? Also same note with `[Tauri]` maybe not linking properly.

Reply:
Thanks that sounds better.